doc/sphinx/advanced_methods.rst

```diff
@@ -397,711 +397,1003 @@ ibm\_triel, ibm\_tribend and ibm\_volCons:
+command::
 
-elastic modulus for volume forces
+    code_info.features()
 
-switch to turn on the computation of local outward normal vectors
+in the simulation script.
 
-The four switches ``ks``, ``kslin``, ``kb`` and ``kal`` set elastic
-parameters for local interactions - ``ks`` for hyperelastic edge
-stiffness, ``kslin`` for linear edge stiffness, ``kb`` for angle
-preservation stiffness and ``kal`` for triangle surface preservation
-stiffness. This stiffness can be either uniform over the whole object,
-or non-uniform. In case of stretching modulus, we can have spring
-stiffness the same for all edges in the whole object, or we can choose
-the value for every edge of the object separately. Analogically, for
-``kslin``, for ``kal`` and ``kb``. Therefore, there are two options for
-setting ``ks``, ``kslin``, ``kal`` and ``kb`` stiffness. Here is the
-explanation for ``ks``:
+Triangulations of elastic objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  **Uniform stiffness:** To set uniform hyperelastic stiffness for all
-   edges in the object, use ``ks``
+| To create an elastic object, we need a triangulation of the surface of
+  this object. Sample triangulations are provided at
+  http://cell-in-fluid.kst.fri.uniza.sk/en/content/oif-espresso. User
+  can create her own in gmsh, salome or any other meshing software. The
+  required format is as follows:
+  
+| The file ``some_nodes.dat`` should contain triplets of floats (one
+  triplet per line), where each triplet represents the :math:`x, y` and
+  :math:`z` coordinates of one node of the surface triangulation. No
+  additional information should be written in this file, so this means
+  that the number of lines equals the number of surface nodes. The
+  coordinates of the nodes should be specified in such a way that the
+  approximate center of mass of the object corresponds to the origin
+  (0,0,0). This is for convenience when placing the objects at desired
+  locations later.
+  
+| The file ``some_triangles.dat`` should also contain triplets of
+  numbers, this time integers. These refer to the IDs of the nodes in
+  the ``some_nodes.dat`` file and specify which three nodes form a
+  triangle together. Please, note that the nodes’ IDs start at 0, i.e.
+  the node written in the first line of ``some_nodes.dat`` has ID 0, the
+  node in the second line, has ID 1, etc.
 
--  **Non-uniform stiffness:** To set non-uniform hyperelastic stiffness,
-   prepare a file with number of lines equal to the number of edges of
-   the triangulation. Each line should contain a real number between 0
-   and 1, so called “weight”. Then call ``ks`` This command reads the
-   weights :math:`weight_i` for each edge and the stiffness for that
-   edge is set to
+Description of sample script
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. math:: ks_i = ksMin * (1 - weight_i) + ksMax*(weight_i)
+The script described in this section is available at
+http://cell-in-fluid.kst.fri.uniza.sk/en/content/oif-espresso. 
```

Comment:
Shouldn’t the file reside in the samples folder?

Reply:
File added to samples, text adapted....